architecture/routing.adoc

```diff
@@ -85,18 +119,34 @@ Edge termination is configured by setting `tls:termination` to `edge` on your ro
     "host": "www.example.com",
     "serviceName": "hello-nginx",
     "tls": {
-        "termination": "edge",
-        "certificate": "<Certificate_File>",
-        "key": "<Key_File>",
-        "caCertificate": "<CaCertificate_File>"
+        "termination": "edge", <1>
+        "certificate": "<cert_text>", <2>
+        "key": "<cert_text>", <3>
+        "caCertificate": "<cert_text>" <4>
      }
 }
 ----
```

Comment:
Yeah, you've described all the things I wrestled with here. :/ They all show actual values, so I believe example+codeblocks are appropriate here (plus, using sidebars would require `+`s at the end of every line to line break, and the white space & tabs wouldn't render).

It's just these cert-related values that are omitted for readability (cuz they'd take up so much space). However, I've now reverted these back to the very to-the-point "CERT TEXT OMITTED FOR READABILITY", as Paul had originally put. It's probably the clearest.

I also updated the callouts (in later sections) to be more clear that you're putting in the contents of the respective files into these spots.